scripts/feature_tests/upgrade/upgrade_common.sh

```diff
@@ -145,7 +145,7 @@ EOF
 
 function manage_node_taints() {
     kubectl get secrets "${CLUSTER_NAME}"-kubeconfig -n "${NAMESPACE}" -o json | \
-    jq -r '.data.value'| base64 -d > /tmp/kubeconfig-"${CLUSTER_NAME}".yaml
+      jq -r '.data.value'| base64 -d > /tmp/kubeconfig-"${CLUSTER_NAME}".yaml
```

Comment:
Changes on this file should be removed. 
This PR seems to have been created on top of your revert of it. 
We need to re-introduce the changes you reverted first.

Reply:
The nits implemented in this PR don't introduce any functional changes. It shouldn't matter if this PR is merged now or after https://github.com/metal3-io/metal3-dev-env/pull/483 is re-introduced.
It's less CI stress to merge this PR now.